### PR TITLE
home page uplift

### PR DESCRIPTION
**Pull request by ATBull81**
# PR Checklist

Please check if your PR fulfills the following requirements
- [x] The title follows our [guidelines](https://github.com/ideacrew/enroll/blob/trunk/CONTRIBUTING.md#commit)
- [x] Tests for the changes have been added (for bugfixes/features), they use let helpers and before blocks
- [ ] For all UI changes, there is cucumber coverage
- [x] Any endpoint touched in the PR has an appropriate Pundit policy. For open endpoints, reasoning is documented in PR and code
- [x] Any endpoint modified in the PR only responds to the expected MIME types.
- [x] For all scripts or rake tasks, how to run it is documented on both the PR and in the code
- [x] There are no inline styles added
- [x] There are no inline javascript added
- [x] There is no hard coded text added/updated in helpers/views/Javascript. New/updated translation strings do not include markup/styles, unless there is supporting documentation
- [x] Code does not use .html_safe
- [x] All images added/updated have alt text
- [x] Doesn’t bypass rubocop rules in any way

# PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature (requires Feature flag)
- [ ] Data fix or migration (inert code, no impact until run)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Dependency updates (e.g., add a new gem or update to a version)

# What is the ticket # detailing the issue?

Ticket: https://www.pivotaltracker.com/story/show/187141967

# A brief description of the changes

Current behavior: Home page needs uplift, new translations, switched to two column format

New behavior: Home page more closely relates to expected view

# Feature Flag

For all new feature development, a feature flag is required to control the exposure of the feature to our end users. A feature flag needs a corresponding environment variable to initialize the state of the flag. Please share the name of the environment variable below that would enable/disable the feature and which client(s) it applies to.

Variable name:

- [ ] DC
- [ ] ME

# Additional Context
Include any additional context that may be relevant to the peer review process.

# AppScan CodeSweep Failure
In the event of a failed check on the AppScan CodeSweep step of our GitHub Actions workflow, please review the False Positive protocol outlined here: appscan_codesweep/CODESWEEP_FALSE_POSITIVES_README.MD

Add all required notes to this section if the failure is a suspected false positive.

Screenshot 2024-07-09 at 9 48 25 PM

Screenshot 2024-07-10 at 12 09 57 PM

Enrollment tile is not corrected yet

### CODE REVIEW

**Comment by ATBull81** [comment]
Had a strange rebasing issue, old PR: https://github.com/ideacrew/enroll/pull/4013

File: app/views/insured/families/home.html.erb [hunk 1]
```diff
@@ -1,58 +1,91 @@
...
+    <div class="pl-2">
+      <% @person.active_employee_roles.each do |employee_role|%>
+        <%= render 'employer_panel', employee_role: employee_role if newhire_enrollment_eligible?(employee_role) %>
       <% end %>
 
+      <%= render 'sep_message', locals: {family_id: @family.id.to_s} if @active_sep.present? %>
+      <h1><%= l10n("insured.my_coverme_header") %></h1>
+
+      <%= render 'right_column' %>
+      <h2><%= enrollments.last.coverage_year %> <%= l10n('coverage').titleize %></h2>
```
Comment (kristinmerbach) [inline app/views/insured/families/home.html.erb:18]: This breaks the page if the family does not have any enrollments, you need to add a conditional around the whole enrollment section to check for the enrollments existance. if no plans exist, don't show anything there.

File: app/views/insured/families/_consumer_brokers_widget.html.erb [hunk 2]
```diff
@@ -1,63 +1,85 @@
...
       <% if current_broker_agency && display_broker_info_for_consumer %>
-        <br/>
-        <span class="word-break">
-          <%= current_broker_agency&.broker_agency_profile&.legal_name&.titleize %>
-        </span>
+        <%= h(link_to l10n("insured.go_to_my_expert"), '', class: 'btn w-100 btn-default') %>
       <% else %>
-        <br/>
-          <%= l10n(".no_selection") %>
+        <%= h(link_to l10n("insured.assign_an_expert"), '', class: 'btn w-100 btn-default') %>
```
Comment (kristinmerbach) [inline app/views/insured/families/_consumer_brokers_widget.html.erb:16]: this link should open the "Help Me Sign Up modal", so should be similar to `<a data-target="#help_with_plan_shopping" data-toggle="modal" style="<%= link_style %>" id="help_me_sign_up" tabindex="0" onkeydown="handleButtonKeyDown(event, 'help_me_sign_up')">`

File: app/views/insured/families/_consumer_brokers_widget.html.erb [hunk 3]
```diff
@@ -1,63 +1,85 @@
...
-  <div class="panel-body">
-    <div class="text-center">
-      <i aria-hidden="true" class="fa fa-user fa-3x"></i>
+    <div class="pr-2 pl-2 pb-2 w-100">
       <% if current_broker_agency && display_broker_info_for_consumer %>
-        <br/>
-        <span class="word-break">
-          <%= current_broker_agency&.broker_agency_profile&.legal_name&.titleize %>
-        </span>
+        <%= h(link_to l10n("insured.go_to_my_expert"), '', class: 'btn w-100 btn-default') %>
```
Comment (kristinmerbach) [inline app/views/insured/families/_consumer_brokers_widget.html.erb:14]: this should go to the `insured/families/brokers` page

File: app/views/insured/families/_qles_panel.html.erb [hunk 4]
```diff
@@ -1,25 +1,36 @@
...
-          <small>
-            <%= l10n("insured.how_may_effect_your_health_insurance") %>
-          </small>
-        </div>
+<% if @bs4 %>
+  <% if EnrollRegistry.feature_enabled?(:show_sep_attestation_options) %>
+    <div class="home-widget p-2">
+      <h3><%= l10n("insured.report_a_life_change") %></h3>
+      <div class="pr-2 pl-2 pb-2 w-100">
+        <%= h(link_to l10n("insured.update_family_information"), manage_family_insured_families_path(tab: 'family'), class: 'btn w-100 btn-default ml-auto pull-right') %>
```
Comment (kristinmerbach) [inline app/views/insured/families/_qles_panel.html.erb:6]: this needs conditional logic per Sarah B. 

> If the user has ever had a draft financial assistance application (regardless of the year), the redirect should be to the applications index.
If the user has never had a financial assistance application, redirect should be to whatever we're renaming the 'Manage Family' page

**Review by kristinmerbach — CHANGES_REQUESTED** [review]
Didn't go too deep into the details, but have some comments on the business logic

File: app/views/insured/families/home.html.erb (hunk 1 above)
Comment (ATBull81) [reply app/views/insured/families/home.html.erb]: wrapped inside an if: https://github.com/ideacrew/enroll/pull/4022/commits/8d2e0ecdcb50420a27185714173553a5e9dc36fd#diff-11d363b5759f355e48892811af2e745d24def1f9b957b65821cead88bf0fc716R18-R24

File: app/views/insured/families/_qles_panel.html.erb (hunk 4 above)
Comment (ATBull81) [reply app/views/insured/families/_qles_panel.html.erb]: wrapped in faa check https://github.com/ideacrew/enroll/pull/4022/commits/8d2e0ecdcb50420a27185714173553a5e9dc36fd#diff-46a4854b30bb71795971cedfd43979141bf17d1a33a4cb3b5d636e53213ec377R6-R11

File: app/views/insured/families/_consumer_brokers_widget.html.erb (hunk 2 above)
Comment (ATBull81) [reply app/views/insured/families/_consumer_brokers_widget.html.erb]: https://github.com/ideacrew/enroll/pull/4022/commits/8d2e0ecdcb50420a27185714173553a5e9dc36fd#diff-9dd370264a65828beb242d7bd2412ed3cb0e855ba1d5c54e99b91004dea7c5d3R16

File: app/views/insured/families/_consumer_brokers_widget.html.erb (hunk 3 above)
Comment (ATBull81) [reply app/views/insured/families/_consumer_brokers_widget.html.erb]: https://github.com/ideacrew/enroll/pull/4022/commits/8d2e0ecdcb50420a27185714173553a5e9dc36fd#diff-9dd370264a65828beb242d7bd2412ed3cb0e855ba1d5c54e99b91004dea7c5d3R14

**Comment by kristinmerbach** [comment]
looks good, but the sep message styling is off. Need something like this added to the _sep_message_html.erb partial:

```ruby
<% if @bs4 %>
  <div class="container mb-0 px-0">
    <div class="alert alert-info d-flex flash align-items-center" role="alert">
      <div class="pl-1">
        <div class="info-icon icon" alt="info">&nbsp;</div>
      </div>
      <div class="col my-1 mr-auto p-0 align-self-center">
        <% if EnrollRegistry.feature_enabled?(:verbose_sep_message) %>
          <div style="color: #323130;">
            <%= l10n("insured.sep_verbose_eligibility_alert", sep_end_date: @active_sep.end_on ) %>
            <span style="text-decoration: underline" class="font-weight-bold font-color-inherit">
              <%= build_link_for_sep_type(@active_sep, nil, local_assigns[:family_id]) %>
            </span>
          </div>
      <% else %>
        <strong>
          <%= l10n("insured.sep_eligibility_alert", sep_end_date: @active_sep.end_on ) %>
          <div class="btn btn-default ml-1">
            <%= build_link_for_sep_type(@active_sep, nil, local_assigns[:family_id]) %>
          </div>
        </strong>
      <% end %>
      </div>
      <div><a class="close-icon icon pr-1 align-self-start" alt="Close" href="#">&nbsp;<span class="sr-only">Close</span></a></div>
    </div>
  </div>
<% else %>
```

**Review by kristinmerbach — APPROVED** [review]
LGTM!

**Review by charlienparker — APPROVED** [review]

**Pull request merged**